desktop-widgets/mainwindow.cpp

```diff
@@ -78,6 +78,8 @@ extern "C" int updateProgress(const char *text)
 	if (progressDialog) {
 		progressDialog->setLabelText(text);
 		progressDialog->setValue(++progressCounter);
+		int width = QFontMetrics(qApp->font()).width(text) + 100;
```

Comment:
i would have used `const int` and also the `font()` member from the `progressDialog` instance.
but tested the same solution and lgtm.